scripts/run.py

```diff
@@ -460,10 +460,10 @@ def process_mission_total():
         'total_climate_change', 'total_climate_change_wc']]
 
     #Create columns to store new data
-    df[["capacity_per_user", "emission_per_capacity", "per_cost_emission", 
-        "per_subscriber_emission", "capex_per_user", "opex_per_user", 
+    df[["capacity_per_user", "per_subscriber_emission", 
+        "capex_per_user", "opex_per_user", "monthly_gb",
         "tco_per_user", "capex_per_capacity", "opex_per_capacity", 
```

Comment:
Where do we use capex_per_capacity? I think this is wrong. Double check and triple check all metrics, removing any redundant/superfluous metrics.

Reply:
Complete.